sceptre/helpers.py

```diff
@@ -117,3 +120,10 @@ def null_context():
     available in py3.6, so providing it here instead.
     """
     yield
+
+
+def get_response_datetime(resp):
```

Comment:
I think this function name and parameter name could definitely use improvement. How about something like:
`extract_datetime_from_aws_response_headers(boto_response)`?

Also, a docstring should definitely be here.

Reply:
Also type annotations would be great.

Reply:
I like it. Will do.